### DISCUSSION TITLE
Welcome to wavesurfer.js Discussions!

### DISCUSSION THREAD

**Discussion by katspaugh** [katspaugh/wavesurfer.js#2670] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by prakharpbuf** [comment]
Hi @katspaugh and Wavesurfer.js community,

I would like to express my appreciation to you, @thijstriemstra, and all the contributors for creating this amazing library! I have been using Wavesurfer.js with React to develop a Digital Audio Workstation from scratch, and it has been an exciting and challenging experience.

As a young professional with a Bachelor's degree in Computer Science and about one year of experience, I am still learning and expanding my skills. However, I have been answering some of the questions posted in this community, and I am enjoying being part of it.

If there are any features or fixes that need to be implemented, please do not hesitate to let me know. I believe in giving back to the community, and I would be thrilled to contribute in any way I can.

Thank you all for being a supportive and open-minded community, and I am looking forward to engaging with you further!
